source/Calamari/Kubernetes/Integration/HelmCli.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Calamari.Common.Features.Processes;
+using Calamari.Common.Plumbing.Logging;
+using Calamari.Common.Plumbing.Variables;
+
+namespace Calamari.Kubernetes.Integration
+{
+    public class HelmCli : CommandLineTool
+    {
+        public HelmCli(ILog log, ICommandLineRunner commandLineRunner, string workingDirectory, Dictionary<string, string> environmentVars)
+            : base(log, commandLineRunner, workingDirectory, environmentVars)
+        {
+            ExecutableLocation = "helm";
+        }
+
+        readonly List<string> builtInArguments = new List<string>();
+
+        public HelmCli WithExecutable(string customExecutable)
+        {
+            ExecutableLocation = customExecutable;
+            return this;
+        }
+
+        public HelmCli WithExecutable(IVariables variables)
```

Comment:
Same logic used in `HelmUpgradeConvention` to get the executable - now with tests!

The only thing that the existing convention does is a `chmod +x` in the script before running it - I'm going to do some testing to make sure that the `CommandLineRunner` doesn't need that to happen before I merge this.

Reply:
I've tested this on linux and it works as well as it does without - there are some permission issues with the existing step if you use helm from a package and it's got the wrong permissions.

I'll ask someone to test it out in windows.

Reply:
Successfully tested on a windows machine with a windows worker: https://octopusdeploy.slack.com/archives/C0431AXP230/p1730072949591379